### ISSUE TITLE
Blender Uploading

### ISSUE THREAD

**Issue by TakahashiEdenyako** [niacdoial/blemd#42]
Yeah, I’m so sorry to bother you again. So everything worked pretty smoothly, I saw the entire Valley of Repose model in Blender, it had only white colors which I presume is just how the modeling works. I decided that I would start to see if I can simply model something into it. I left a cube at the home base where the onions and ship start at. I don’t know how to save the file however. I clicked on file>export> Then I saw my documents, I clicked on Pikmin2’s root to see if I could export it in the tutorial file. But then it just had the screen where the files were Users>Kando>Map>tutorial and overwrite its arc.szs and texts.szs, but there were no files on it. I tried to save it/save as over in Lunaboy’s folder where I have the model that I can open in blender I clicked on that, but it didn’t let me save it into a folder or something. Anyways, I don’t know if this is something that has to do with the bmd.importer and why it won’t let me (which I’m thinking would work) overwrite the Valley o R’s model.

**Comment by niacdoial** [comment]
I have no idea what you tried to do to export the model, but it's sadly impossible.
For now, BleMD can only import BMD files into blender, not export to BMD files.
I hope this isn't too disappointing for you.

**Comment by TakahashiEdenyako** [comment]
Can I save it anyhow?

**Comment by niacdoial** [comment]
you can save it as a blender file.
or, if you find something that can export to BMD files, you might want to look at what that other tool can import (fbx, obj, other extensions like that. Blender can export those.)

**Comment by TakahashiEdenyako** [comment]
![Screenshot (3)](https://user-images.githubusercontent.com/85026028/120117347-cc13d000-c15a-11eb-8092-df9db17cda9f.png)

I clicked save, but I don't see where to save it as a blender file.

**Comment by niacdoial** [comment]
wherever you want, really. Or you could put it next to the BMD file I guess.
go the the folder you want, just change "untitled.blend" to the name you want, and click save.

**Comment by TakahashiEdenyako** [comment]
I tried doing that, but there's no save button. Also, really sorry it's in Japanese, it was the only video I could find for what I'm trying to do and it in the left bottom corner is a box where it says the tools they used.

**Comment by niacdoial** [comment]
I… 
top-right. "Save Blender File". I don't want to be rude or anything, and I know that the blender 2.7x interface is cluttered, but like. it was right there? in plain sight?

also, what Japanese? I don't see anything.
…
wait you edited the previous comment (and assumed I saw that and read your comment again). let me look.

**Comment by niacdoial** [comment]
OK.
from blender you should just use "export->DAE" (or "export->openCollada"? I can't remember), then use the SuperBMD thing on this DAE file.

**Comment by TakahashiEdenyako** [comment]
So those are dowloads like that I get on Github? Do i do the same process where I put their file into Blender's script and addons?

**Comment by niacdoial** [comment]
no, SuperBMD seems like a program that runs on its own. Didn't you see that the one who made that video just dragged a DAE file into a "SuperBMD.exe" program?
and to find it, I'ld say either https://github.com/RenolY2/SuperBMD or https://github.com/Sage-of-Mirrors/SuperBMD.

**Comment by TakahashiEdenyako** [comment]
It doesn't have the same thing in the video where it says just export. There doesn't appear to be a DAE.
![Screenshot (4)](https://user-images.githubusercontent.com/85026028/120118045-717c7300-c15e-11eb-82a0-6a3be3189252.png)

**Comment by niacdoial** [comment]
it's probably an add-on you can enable

**Comment by TakahashiEdenyako** [comment]
Wait, I'm so sorry, I forgot about the collada option for a second.

**Comment by TakahashiEdenyako** [comment]
![Screenshot (5)](https://user-images.githubusercontent.com/85026028/120118086-b6a0a500-c15e-11eb-9d11-159663b9c574.png)

This is after i clicked collada, but where is the DAE file and what is that?

**Comment by niacdoial** [comment]
I… OK I guess

also, you are *creating* the DAE file, so it doesn't exist yet.  That is the meaning of "exporting to" DAE.

**Comment by TakahashiEdenyako** [comment]
Okay, I think I'm a little bit on a clearer path with what I just did. Now I don't fully understand how github files. So, I exported the file into my folder and draged it to the folder (not the exe.) but the folder with the SuperBMD files. Do you know where I drag the DAE to?
![Screenshot (6)](https://user-images.githubusercontent.com/85026028/120118216-7aba0f80-c15f-11eb-9195-e3ccc2a54223.png)

**Comment by niacdoial** [comment]
oh yeah.
the way you downloaded superBMD, it doesn't have the exe file ready, and it needs to be created.
try downloading SuperBMD from here: https://github.com/RenolY2/SuperBMD/releases/download/v2.1.0/SuperBMD.zip

**Comment by TakahashiEdenyako** [comment]
Alright, I downloaded that and dragged the DAE into the exe, What does this mean?
![Screenshot (7)](https://user-images.githubusercontent.com/85026028/120118394-5448a400-c160-11eb-92f2-fe59450620cd.png)

**Comment by niacdoial** [comment]
long story short, it means you need to start over and import the BMD file into blender, but this time check the option called "Force create bones".

**Comment by TakahashiEdenyako** [comment]
Aight, I’ll be back in a minute😎

**Comment by TakahashiEdenyako** [comment]
![Screenshot (8)](https://user-images.githubusercontent.com/85026028/120118707-e69d7780-c161-11eb-85d3-ba8b497cf05d.png)
![Screenshot (9)](https://user-images.githubusercontent.com/85026028/120118709-e69d7780-c161-11eb-820f-3363ccb9dff4.png)
![Screenshot (9)](https://user-images.githubusercontent.com/85026028/120118714-e9986800-c161-11eb-82e2-7c830f51f956.png)


Alright, now i dragged it to SuperBMD, I don't know if that's where I drag it.

**Comment by TakahashiEdenyako** [comment]
and i turned on the bones

**Comment by niacdoial** [comment]
so, SuperBMD opened and closed?
check if a BMD file appeared next to the DAE file, or if the BMD file from before was overwritten.

**Comment by TakahashiEdenyako** [comment]
The DAE file that's in that foler I screenshot? Next to, do you litterally mean left or right to, or up and down to? If it's in that file up and down, I couldn't fine any BMD next to it or in the folder.

**Comment by niacdoial** [comment]
1: yes, "next to" also meant up or down.   when I said "next to", it was to say "in the same folder".
2: have you re-run SuperBMD with the new DAE file? I should have said that, maybe

**Comment by TakahashiEdenyako** [comment]
Re-run means to drag it into SuperBMD, right? I did that, and it said...
![Screenshot (10)](https://user-images.githubusercontent.com/85026028/120119129-fa49dd80-c163-11eb-9a22-f214c4e806db.png)


I was absoulutely sure to put skelelton on. I really did. Do I have to save the blender file maybe? I just thought that this time because I knew what to do to make a dae that i wouldnt have to save a blender file.

**Comment by niacdoial** [comment]
you sure you dragged the right file into SuperBMD?
if you did, I have no idea what happened, sorry.
I need to go sleep now. See you.

**Comment by TakahashiEdenyako** [comment]
I’ll figure it out tomorrow. Good night and thanks for helping me.

**Comment by niacdoial** [comment]
Since you didn't reply since that day, I'll have to assume things worked!
Good for you!
If I'm wrong, feel free to reopen this.
